crates/weaver_semconv/Cargo.toml

```diff
@@ -16,6 +16,7 @@ weaver_common = { path = "../weaver_common" }
 
 serde.workspace = true
 serde_yaml.workspace = true
+serde_json.workspace = true
```

Comment:
If JSON is only used in unit tests, which I believe is the case, then you should move this dependency to the `[dev-dependencies]` section.